test/evergreen.yml

```diff
@@ -216,16 +216,18 @@ functions:
           set -o errexit
           set -o verbose
 
-          # Check whether specific branches have been provided to the function
-          if [ -z ${branches} ]; then
-            wt_branches=${branch_name}
+          # Check whether specific branches have been provided to the function through the task
```

Comment:
whether --> if
have been --> are

Reply:
Thanks, corrected in ed593ccc050d835888324ecda4396e4f18c41316